velox/functions/lib/DateTimeFormatter.cpp

```diff
@@ -2020,9 +2020,6 @@ Expected<std::shared_ptr<DateTimeFormatter>> buildSimpleDateTimeFormatter(
         case 'D':
           builder.appendDayOfYear(count);
           break;
-        case 'u':
-          builder.appendDayOfWeek1Based(count);
-          break;
```

Comment:
Keep alphabetical order

Reply:
Maybe it placed here because the next is appendDayOfWeekText, both way can be acceptable, integrate with appendDayOfWeekxxx or alphabetical order